samples/apply-blend-renderer-to-hillshade/src/main/java/com/esri/arcgismaps/sample/applyblendrenderertohillshade/screens/ApplyBlendRendererToHillshadeScreen.kt

```diff
@@ -0,0 +1,211 @@
+                        description = messageDescription,
+                        onDismissRequest = ::dismissDialog
+                    )
+                }
+            }
+        }
+    }
+}
+
+@Composable
+private fun DialogOptions(
+    altitude: Double,
+    onAltitudeChange: (Double) -> Unit,
+    azimuth: Double,
+    onAzimuthChange: (Double) -> Unit,
+    slopeTypeOptions: List<Pair<String, com.arcgismaps.raster.SlopeType?>>,
+    selectedSlopeType: com.arcgismaps.raster.SlopeType?,
+    onSlopeTypeSelected: (com.arcgismaps.raster.SlopeType?) -> Unit,
+    colorRampPresets: List<String>,
+    selectedColorRampIndex: Int,
+    onColorRampSelected: (Int) -> Unit,
+    onDismissRequest: () -> Unit
+) {
+    SampleDialog(onDismissRequest = onDismissRequest) {
+        Text("Renderer Settings", style = MaterialTheme.typography.titleMedium)
+
+        // Slope type dropdown
+        val slopeLabels = slopeTypeOptions.map { it.first }
+        val selectedSlopeIndex = slopeTypeOptions.indexOfFirst { it.second == selectedSlopeType }.coerceAtLeast(0)
+        DropDownMenuBox(
+            textFieldValue = slopeLabels.getOrNull(selectedSlopeIndex) ?: slopeLabels[0],
+            textFieldLabel = "Slope Type",
+            dropDownItemList = slopeLabels,
+            onIndexSelected = { index -> onSlopeTypeSelected(slopeTypeOptions[index].second) }
+        )
+
+        // Color ramp dropdown
+        DropDownMenuBox(
+            textFieldValue = colorRampPresets.getOrNull(selectedColorRampIndex) ?: colorRampPresets[0],
+            textFieldLabel = "Color Ramp Preset",
+            dropDownItemList = colorRampPresets,
+            onIndexSelected = onColorRampSelected
+        )
+
+        // Altitude slider
+        Column(modifier = Modifier.fillMaxWidth(), verticalArrangement = Arrangement.spacedBy(6.dp)) {
+            Text("Altitude: ${altitude.toInt()}°")
+            Slider(
+                value = altitude.toFloat(),
+                onValueChange = { onAltitudeChange(it.toDouble()) },
+                valueRange = 0f..360f
+            )
+        }
+
+        // Azimuth slider
+        Column(modifier = Modifier.fillMaxWidth(), verticalArrangement = Arrangement.spacedBy(6.dp)) {
+            Text("Azimuth: ${azimuth.toInt()}°")
+            Slider(
+                value = azimuth.toFloat(),
+                onValueChange = { onAzimuthChange(it.toDouble()) },
+                valueRange = 0f..360f
+            )
+        }
+
+        // Dismiss button row
+        Row(
+            modifier = Modifier.fillMaxWidth(),
+            horizontalArrangement = Arrangement.End,
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            OutlinedButton(onClick = onDismissRequest, modifier = Modifier.wrapContentWidth()) {
+                Text("Dismiss")
+            }
+            Button(onClick = onDismissRequest, modifier = Modifier.padding(start = 8.dp)) {
+                Text("Done")
+            }
```

Comment:
Since changes are applied to the MapView in real-time, we don't need two buttons to close the dialog here: 
```suggestion
            OutlinedButton(onClick = onDismissRequest, modifier = Modifier.wrapContentWidth()) {
                Text("Done")
            }
```